FirebaseSwiftUI/README.md

```diff
@@ -0,0 +1,139 @@
+4. Select target(s) you wish to add to your app (currently `FirebaseAuthSwiftUI`, `FirebaseGoogleSwiftUI`, `FirebaseFacebookSwiftUI` and `FirebasePhoneAuthSwiftUI` are available). `FirebaseAuthSwiftUI` is required and contains Email provider API.
+5. Press `Add Packages` button to complete installation.
+
+
+## Getting started
+
+1. Follow step 2, 3 & 5 on [adding Firebase to your SwiftUI app](https://firebase.google.com/docs/ios/setup).
+2. You should now update your app entry point to look like this:
+
+```swift
+import FirebaseAuthSwiftUI
+import FirebaseCore
+import SwiftUI
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [
+                     UIApplication.LaunchOptionsKey: Any
+                   ]?) -> Bool {
+    FirebaseApp.configure()
+    return true
+  }
+}
+
+@main
+struct FirebaseSwiftUIExampleApp: App {
+  @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+
+  init() {}
+
+  var body: some Scene {
+    WindowGroup {
+        ContentView()
+    }
+  }
+}
+
+struct ContentView: View {
+  let authService: AuthService
+
+  init() {
+    let configuration = AuthConfiguration()
+
+    authService = AuthService(
+      configuration: configuration,
+    )
+    .withEmailSignIn()
+  }
+
+  var body: some View {
+    AuthPickerView().environment(authService)
+  }
+}
+```
+
+3. For a more complete example, see the [SwiftUI sample app](https://github.com/firebase/FirebaseUI-iOS/tree/main/samples/swiftui/FirebaseSwiftUIExample).
+
+## Configuration options
+
+You can create an `AuthConfiguration` instance and pass it to the `AuthService` as demonstrated above. Here are the options:
+
+```swift
+public struct AuthConfiguration {
+  // hides cancel buttons when you don't want a flow to be interrupted
+  let shouldHideCancelButton: Bool
+  // stop users from being able to swipe away sheets/modal
+  let interactiveDismissEnabled: Bool
+  // automatically upgrade anonymous users so that they are linked with account being used to sign-in
+  let shouldAutoUpgradeAnonymousUsers: Bool
+  // custom string bundle for string localizations
+  let customStringsBundle: Bundle?
+  // terms of service URL
+  let tosUrl: URL
+  // privacy policy URL
+  let privacyPolicyUrl: URL
+  // action code settings for email sign in link
+  let emailLinkSignInActionCodeSettings: ActionCodeSettings?
+  // action code settings verifying email address
+  let verifyEmailActionCodeSettings: ActionCodeSettings?
+
+  public init(shouldHideCancelButton: Bool = false,
+              interactiveDismissEnabled: Bool = true,
+              shouldAutoUpgradeAnonymousUsers: Bool = false,
+              customStringsBundle: Bundle? = nil,
+              tosUrl: URL = URL(string: "https://example.com/tos")!,
+              privacyPolicyUrl: URL = URL(string: "https://example.com/privacy")!,
```

Comment:
Maybe these parameters shouldn't have default values, or should be nullable. It would be an error to ship your app with these default values.

(This can be addressed in a separate PR)

Reply:
Good point - I think nullable is right. I'll address in separate PR.